recipes/wagtail/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "wagtail" %}
+{% set version = "1.11.1" %}
+{% set sha256 = "fb54795d91c79cbaacb40ccfdbf047fa31a49fb65f57ec205d3656af4a9372b7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
Please add an `entry_points` section for:

https://github.com/wagtail/wagtail/blob/master/setup.py#L101-L103

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points